Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Views/EntryDelegate.cpp

```diff
@@ -113,7 +111,6 @@ namespace AzToolsFramework
                         // sources with no children should be greyed out.
                         if (sourceEntry->GetChildCount() == 0)
                         {
-                            isEnabled = false; // draw in disabled style.
                             actualPalette.setCurrentColorGroup(QPalette::Disabled);
```

Comment:
I think `actualPallete` is no longer used now either and I'm surprised it isn't causing an unused var error

Reply:
The reason that `actualPalette` variable doesn't cause an unused variable warning is that the compiler can only determine variable usage for the function it is compiling.
Since a member function is called using `actualPalette` variable, the compiler has to assume that there are side effects to calling other functions that might affect global state(such as accessing as the function calling `QCoreApplication::instance()` and changing the QT state)`